pkg/api/api.go

```diff
@@ -119,7 +120,11 @@ func New(port string, manager *capture.Manager, opts ...Option) (*Server, error)
 	}
 
 	// initialize currently supported APIs
-	v1Options := []v1.Option{v1.WithLogger(s.logger)}
+	v1Options := []v1.Option{
+		v1.WithLogger(s.logger),
+		// inject standard error handler and attach api logger
+		v1.WithErrorHandler(errors.NewStandardHandler(s.logger)),
```

Comment:
I don't get the additional advantage `pkg/api/errors` comes with. How does this pkg enriche the error information?
Also, `WithLogger` and `WithErrorHandler` are both initialized with the same `log.Logger`.

Reply:
It enriches the error information in the sense that it returns a concise error message to the caller and logs a detailed one on the server.

It is by design that both the API logger and the ErrorHandler have the same logger.